packages/web-react/src/components/Alert/useAlertIcon.tsx

```diff
@@ -0,0 +1,14 @@
+import { useIconName } from '../../hooks/useIconName';
+import { SpiritAlertProps } from '../../types';
+
+export function useAlertIcon(props: SpiritAlertProps) {
+  const { color, iconName } = props;
+  const iconNameValue = useIconName(color, {
+    default: 'info',
+    danger: 'warning',
+  });
+
+  return {
```

Comment:
You are returning only one value, so outputting an object is not necessary here.

```suggestion
  return iconName || iconNameValue;
```